role-service/src/main/java/uk/ac/ox/ndph/mts/role_service/controller/dtos/PermissionDTO.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ox.ndph.mts.role_service.controller.dtos;
+
+
+import javax.validation.constraints.NotBlank;
+
+
+public class PermissionDTO {
+
+    @NotBlank (message = "Permission ID cannot be empty")
+    private String id;
```

Comment:
Should this also include a max notation like you did in the Role one?

Reply:
Thanks @tamirkamara .  My initial thinking was that since there is no create-permission endpoint (and the liquibase load won't go through this filter)  then there was no point in any validation here on these IDs -  other than a not-empty check to ensure downstream code can lookup by ID.   However this is probably over-thinking it, and at some point we may extend this to enable permission creation.  I've added the validation in to match the DB field.